cmd/clusterctl/clusterdeployer/clusterclient/clusterclient.go

```diff
@@ -166,17 +166,16 @@ func (c *client) EnsureNamespace(namespaceName string) error {
 }
 
 func (c *client) GetKubeconfigFromSecret(namespace, clusterName string) (string, error) {
```

Comment:
Would it be worth it to move this method to `util/kubeconfig/kubeconfig.go` as well?

Reply:
We'd still need the actual method here for clusterctl to use, but +1 to having a similar one in the util package

Reply:
Is this something I should incorporate to this PR, or maybe do as a follow-up PR?

Reply:
Followup is good.

Reply:
Captured this in https://github.com/kubernetes-sigs/cluster-api/issues/1306